aliases/available/node.aliases.bash

```diff
@@ -0,0 +1,8 @@
+cite 'about-alias'
+about-alias 'the nodejs environment aliases'
```

Comment:
Please change to "Node.js", which is the official name...

Reply:
Done